src/plugins/discover/public/application/components/doc/use_es_doc_search.ts

```diff
@@ -33,7 +38,8 @@ export function buildSearchBody(id: string, indexPattern: IndexPattern): Record<
       },
     },
     stored_fields: computedFields.storedFields,
-    _source: true,
+    _source: !useNewFieldsApi,
+    fields: useNewFieldsApi ? ['*'] : undefined,
```

Comment:
Will this take into account the field filters? And we still won't show properly formatted `date_nanos` if we're explicitly not requesting a list of date fields with the appropriate format like in the `search_source`?

Reply:
@majagrubic we currently don't have index pattern information here, it's just a simple request to ES. So making this would be an enhancement